code/src/qik_2s12v10.cpp

```diff
@@ -2,20 +2,12 @@
 
 namespace r2d2::moving_platform {
 
-    const uint8_t qik_2s12v10_c::qik_autodetect_baudrate = 0xAA;
-    const uint8_t qik_2s12v10_c::qik_request_firmwareversion = 0x81;
-    const uint8_t qik_2s12v10_c::qik_motor_m0_set_forward = 0x88;
-    const uint8_t qik_2s12v10_c::qik_motor_m0_set_reverse = 0x8A;
-    const uint8_t qik_2s12v10_c::qik_motor_m1_set_forward = 0x8C;
-    const uint8_t qik_2s12v10_c::qik_motor_m1_set_reverse = 0x8E;
-    const uint8_t qik_2s12v10_c::qik_get_config_parameter = 0x83;
-    const uint8_t qik_2s12v10_c::qik_get_error = 0x82;
 
     qik_2s12v10_c::qik_2s12v10_c(r2d2::uart_ports_c &uart_port,
-                                 unsigned int baudRate,
+                                 unsigned int baud_rate,
                                  hwlib::pin_out &reset_pin): 
         reset_pin(reset_pin), 
-        usart_bus(baudRate, uart_port) 
+        usart_bus(baud_rate, uart_port) 
         {}
 
     void qik_2s12v10_c::set_speed(const int8_t &new_speed) {
```

Comment:
` void qik_2s12v10_c::set_speed(const int8_t &new_speed) {` 
const & is unnecessary to use, it cost more memory and is slower. Also in style guide

should be: 
` void qik_2s12v10_c::set_speed(int8_t new_speed) {`

It will be loads of errors, so many other files should be adjusted. Cost maybe too much time